hedera-mirror-rest/__tests__/transactionResult.test.js

```diff
@@ -0,0 +1,53 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+'use strict';
+
+const proto = require('@hashgraph/proto/lib/proto');
```

Comment:
Can simplify the import so you don't have `proto.proto`:
```suggestion
const {proto} = require('@hashgraph/proto/lib/proto');
```

Reply:
Good call, done.

Reply:
actually we should do:

```javascript
const proto = require('@hashgraph/proto');
```

then use `proto.ResponseCodeEnum`. The `lib/proto` is not exported and subject to change.